README.md

```diff
@@ -146,5 +146,14 @@ $ nix-build
 
 If you want to test **buildMavenRepository** you can run:
 ```bash
-$ nix-build -A buildMavenRepositoryFromLockFile --arg "./mvn2nix-lock.json"
+$ nix-build -A buildMavenRepositoryFromLockFile --arg file "./mvn2nix-lock.json"
 ```
+
+At the moment, _mvn2nix_ is **self-bootstrapped** consuming it's own _mvn2nix-lock.json_.
+If for whatever reason, the JSON format changes or you need to update the dependencies, you can run _mvn2nix-bootstrap_.
+
+```bash
+$ nix-build -A mvn2nix-bootstrap
+
+./result/bin/mvn2nix > example/mvn2nix-lock.json
+```
```

Comment:
@roberth makes sense; here is the relevant "escape hatch" to build it in the non mvn2nix way.
I like having the typical build though "bootstrapped" because it signals to users how to use the package.

Reply:
Ah, so they may have to change the output hash in that drv, but otherwise only relies on maven.
Doing this in the sandbox is a bit unusual. Is maintaining the output hash worth it?